artemis/src/main/java/com/artemis/EntityManager.java

```diff
@@ -181,7 +179,7 @@ void synchronize(EntitySubscription es) {
 		for (int i = 0; i < entities.size(); i++) {
 			Entity e = entities.get(i);
 			if (e != null)
-				es.check(e.id);
+				es.check(e.getId());
```

Comment:
`Entity#id` is package local though - it's mostly about disallowing users from updating the id.